### PR TITLE
chore(session-analysis): add session analytics and session warning

### PR DESCRIPTION
**Pull request by rcmarron**
## Problem

We don't know if people are using session analysis + they might be confused why events are missing

## Changes

Adds a 'usingSessionAnalysis' property to `insight viewed` and `insight analyzed`

Adds a banner telling people events without sessions won't be tracked if they're using session analysis ([Figma](https://www.figma.com/file/gQBj9YnNgD8YW4nBwCVLZf/PostHog-App?node-id=2207%3A16554))

image

Note: The docs link will get way better once [this PR](https://github.com/PostHog/posthog.com/pull/3736) is in

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?
 Wrote logic tests and verified the banner + event property appear if:
* Using unique session
* Using session duration as an event math property
* Using session duration as an entity property filter
* Using session duration as a global property filter
* Using session duration as a breakdown

### CODE REVIEW

File: frontend/src/lib/utils/eventUsageLogic.ts [hunk 1]
```diff
@@ -528,13 +531,7 @@ export const eventUsageLogic = kea<eventUsageLogicType>({
...
             properties.total_event_actions_count = (properties.events_count || 0) + (properties.actions_count || 0)
 
-            let totalEventActionFilters = 0
-            const entities = (filters.events || []).concat(filters.actions || [])
-            entities.forEach((entity) => {
-                if (entity.properties?.length) {
-                    totalEventActionFilters += entity.properties.length
-                }
-            })
+            const totalEventActionFilters = 0
```
Comment (neilkakkar) [inline frontend/src/lib/utils/eventUsageLogic.ts:534]: why this change? 🤔

Comment (rcmarron) [reply frontend/src/lib/utils/eventUsageLogic.ts]: Ooooo good catch! That shouldn't be there - sloppy commits (had started a refactor and then decided against it)

**Review by neilkakkar — APPROVED** [review]
